scripts/scil_convert_gradients_mrtrix_to_fsl.py

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+
+from scilpy.io.utils import (assert_inputs_exist, assert_outputs_exist,
+                             add_overwrite_arg)
+from scilpy.utils.bvec_bval_tools import mrtrix2fsl
+
+DESCRIPTION = "Script to convert bval/bvec MRtrix style to FSL style."
+
+
+def _build_args_parser():
+    p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                description=DESCRIPTION)
+
+    p.add_argument('mrtrix_enc',
+                   help='Gradient directions encoding file. (.b)')
+
+    p.add_argument('fsl_basename',
```

Comment:
I personally prefer to have bval and bvec argument separated. With this it is mandatory to have .bval and .bvec